### DISCUSSION TITLE
HACS

### DISCUSSION THREAD

**Discussion by gideonklein** [indomus/forum#454] [HUB: Home Assistant]
Ciao a tutti, hacs funziona bene ma nella configurazione .yaml lo dà come errore. Allego lo screen
Qualcuno sa aiutarmi? 
![Cattura](https://user-images.githubusercontent.com/79428290/123050978-8ff22900-d401-11eb-9483-9655336f4e8b.PNG)

**Comment by NdR91** [comment]
Ti da errore HA nel check, o solo l'editor?

Va bene così comunque, se funziona tutto magna tranquillo.
Oppure, al limite, passa alla configurazione di HACS via GUI

**Reply by gideonklein** [reply]
Grazie, non dava nessun problema nonostante l'errore. Comunque ho disinstallato tutto e fatto una nuova installazione (https://hacs.xyz/docs/configuration/start/) ora è sulle integrazione e cancellato la stringa con il token su Yaml
